### DISCUSSION TITLE
[Fundamentos do C#] [AULA:  Escopo de um programa] Em Program.cs o código inicial atual é diferente do código inicial do curso

### DISCUSSION THREAD

**Discussion by ghost** [balta-io/forum#416] [Tira Dúvidas] [answered]
Assim que crio um arquivo C# em versões do .NET 6 em diante, o modelo de projeto para novos aplicativos de console do C# gera o seguinte código no arquivo Program.cs:

`// See https://aka.ms/new-console-template for more information`
`Console.WriteLine("Hello, World!");`

No Curso de Fundamentos do C#, em grande parte dos módulos o código inicial vem assim:

`using System;`
`namespace MeuApp // Nome do projeto`
`{`
`    class Program`
`    {`
`        static void Main(string[] args)`
`        {`
`            Console.WriteLine("Hello World!");`
`        }`
`    }`
`}`

> O que muda do primeiro código para o segundo? Eu posso utilizar o primeiro?

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @BrunoZancan, como vai?

Na versão 6 do .NET houve uma mudança no programa inicial, você pode usar a flag `--use-program-main` para manter igual do curso caso esteja utilizando uma versão mais nova do .NET.

Por exemplo:
```
dotnet new console -o MeuProjeto --use-program-main
```

**Reply by ghost** [reply]
Bom dia, muito obrigado!
